charts/greptimedb-operator/templates/clusterrole.yaml

```diff
@@ -78,6 +79,7 @@ rules:
       - create
       - get
       - list
+      - patch
```

Comment:
Make the policy same as this: https://github.com/GreptimeTeam/greptimedb-operator/blob/main/config/rbac/role.yaml